### ISSUE TITLE
Convert Schrodinger JACS benchmark sets to PDB as examples?

### ISSUE THREAD

**Issue by jchodera** [choderalab/yank#227]
Look into converting Schrödinger JACS benchmark sets.  Check SI from paper.

**Comment by bas-rustenburg** [comment]
https://drive.google.com/folderview?id=0BylmDElgu6QLTnJ2WGMzNXBENkk

**Comment by jchodera** [comment]
Simple conversion script (unfortunately only to mol2):
```python
from schrodinger import structure # Requires Schrodinger Suite

reader = structure.StructureReader("p38_mapper_20142_pv.mae")
writer = structure.StructureWriter("p38_mapper_20142_pv.mol2")
for st in reader:
    writer.append(st)
reader.close()
writer.close()
```

**Comment by bas-rustenburg** [comment]
You could go to pdb format using openeye?
```python
#!/usr/bin/env python
from openeye.oechem import *

ifs = oemolistream()
ofs = oemolostream()

if ifs.open("p38_mapper_20142_pv.mol2"):
    if ofs.open("p38_mapper_20142_pv.pdb"):
        for mol in ifs.GetOEGraphMols():
            OEWriteMolecule(ofs, mol)
    else:
        OEThrow.Fatal("Unable to create 'p38_mapper_20142_pv.pdb'")
else:
    OEThrow.Fatal("Unable to open 'p38_mapper_20142_pv.mol2'")
```

**Comment by kyleabeauchamp** [comment]
It was also easy enough to do this from within Maestro, which allows you to save to either PDB or mol2 and write each ligand to separate files.

**Comment by bas-rustenburg** [comment]
I remember some issues I had when I was working on the LDH stuff. Maestro
was giving weird errors/warnings when saving stuff as pdb files. Did you
try this just now? Just wondering whether they actually prevent you from
converting or not.

**Comment by kyleabeauchamp** [comment]
I just tried it.  It seems OK.  The real test will be running it through PDBFixer and Yank.

**Comment by jchodera** [comment]
Maybe we can convert the receptor to PDB and the ligands to a multimolecule
mol2 and stick this in a new github repo?

**Comment by kyleabeauchamp** [comment]
Yes, we might even call the new repo YankPaper, as hopefully this calculation will be part of the Yank paper.

However, I'd say let's use one frame per mol2, as IMHO it simplifies workflows.

**Comment by jchodera** [comment]
Works for me.

yank-manuscript/datasets/schrodinger-jacs?

Separate files is fine.

**Comment by jchodera** [comment]
Let's just make sure we can match up identifiers of the mol2 files with the
included CSV or pkl database.

**Comment by jchodera** [comment]
Let me know if you guys want me to do this---otherwise, maybe someone can claim who is going to set this repo up?

**Comment by kyleabeauchamp** [comment]
https://github.com/choderalab/Yank-Manuscript

**Comment by Lnaden** [comment]
Do we still want to covert this repo?

1.0 touch?
